.changelog/2428.txt

```diff
@@ -0,0 +1,8 @@
+```release-note:feature
+plugin/k8s: Allows advanced users to add sidecar containers to apps using the k8s plugin config.
```

Comment:
Nitpick but I don't think we need to say "advanced users" here, just "Allows users to".